src/rpc/misc.cpp

```diff
@@ -44,8 +44,8 @@ static UniValue validateaddress(const JSONRPCRequest& request)
             "}\n"
                 },
                 RPCExamples{
-                    HelpExampleCli("validateaddress", EXAMPLE_ADDRESS) +
-                    HelpExampleRpc("validateaddress", EXAMPLE_ADDRESS)
+                    HelpExampleCli("validateaddress", EXAMPLE_ADDRESS[1]) +
```

Comment:
The quotation marks around the address are missing. Previously they were included in `EXAMPLE_ADDRESS`, now as the constant contains the bare address, so they have to be added here.